packages/vuetify/build/rollup.config.mjs

```diff
@@ -32,7 +32,7 @@ export default [
     input: 'src/entry-bundler.ts',
     output: [
       {
-        file: 'dist/vuetify.esm.js',
+        file: 'dist/vuetify.mjs',
```

Comment:
This is a breaking change. The publint warning is meaningless, you would never need to import this.

Reply:
ok, reverting those changes...